cypress/.eslintrc.js

```diff
@@ -1,12 +1,8 @@
 module.export = {
   root: true,
-  plugins: [
-    "eslint-plugin-cypress"
-  ],
-  extends: [
-    "plugin:cypress/recommended"
-  ],
+  plugins: ["eslint-plugin-cypress"],
+  extends: ["plugin:cypress/recommended"],
   env: {
-    "cypress/globals": true
+    "cypress/globals": true,
   },
 };
```

Comment:
Ran yarn format so alot of this is just the formatting running over the project directory. First times a doozy..